app/javascript/components/App.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Greeting from "./Greeting";
+// import HelloWorld from './HelloWorld'
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.